### DISCUSSION TITLE
sophos SSL VPN局域网

### DISCUSSION THREAD

**Discussion by snail0019** [v2fly/v2ray-core#1491] [Q&A]
Hello I was in Europe always by using **Sophos VPN** to connect the local network to access our company file. Now I am in china, I have to use V2Ray to bypass the _great wall_, and then access the company. But Sophos VPN seems not work together with V2ray, any idea? THX
你好，我平时在海外用Sophos来登录海外公司的内网。现在回国，用如何设置V2Fly 可以让我的Sophos联上公司的内网。谢谢

**Comment by Kylejustknows** [comment]
先做好梯子后，本地的v2客户端开启一个本地的http或者socks5代理，然后在sophos里配置使用本地代理上网就行了。

上面的第一句话如何实现，你可以自己搭建，也可以购买，就不多说了。顺便恭喜回国成功！
